docs/conf.py

```diff
@@ -107,6 +110,7 @@
     "navbar_start": ["navbar-logo"],
     "navbar_end": ["navbar-icon-links", "theme-switcher"],
     "navbar_align": "content",
+    "header_links_before_dropdown": 8,
```

Comment:
This was 10 in the gallery. Is this by design?

Reply:
I think 8 is by definition no ``More`` Dropdown